File: backend/src/main/java/cohappy/backend/model/dto/request/PatchDebtPaidDTO.java
package cohappy.backend.model.dto.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class PatchDebtPaidDTO {
    private String debtId;
    private String receiverCode;
    private Boolean newState;
}

File: frontend/app/src/main/java/cohappy/frontend/client/dto/response/PatchDebtPaidDTO.kt
package cohappy.frontend.client.dto.response
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class PatchDebtPaidDTO (
    var debtId: String? = null,
    var receiverCode: String? = null,
    var newState: Boolean? = null
)